queries.sql

```diff
@@ -17,3 +17,32 @@ SELECT * FROM animals WHERE name <> 'Gabumon';
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+-- How many animals are there? 10
+UPDATE animals SET weight_kg = weight_kg * (-1) WHERE weight_kg < 0;
+
+-- How many animals have never tried to escape?
+SELECT COUNT(escape_attempts)
+FROM animals
+WHERE escape_attempts = 0;
+
+-- What is the average weight of animals?
+SELECT COUNT(escape_attempts)
+FROM animals
+WHERE escape_attempts = 0;
```

Comment:
- [ ] Kindly here you have to calculate the average weight of animals, not the number of animals has never tried to escape, you made this query already above.